src/tox_uv/_venv.py

```diff
@@ -310,6 +311,18 @@ def _py_info(self) -> PythonInfo:  # pragma: win32 no cover
             extra={},
         )
 
+    def package_root(self) -> Path:
+        try:
+            return self.core["package_root"]
+        except KeyError:
+            config = self.core["tox_root"] / "tox.ini"
```

Comment:
This can be a lot more, than just ini, like toml and various other file sources we support.